CHANGELOG.md

```diff
@@ -1 +1,2 @@
 * Adds the ability to select an extension to install from a list of available official extensions when `firebase ext:install -i` or `firebase ext:install --interactive` is run.
+* Release RTDB Emulator v4.3.1.
```

Comment:
Does this fix a bug we can reference?  Also nit: replace RTDB with database.

Reply:
Updated wording. No external bug that we can reference though.